CHANGELOG.md

```diff
@@ -3,6 +3,72 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.0.1
+_Mar 28, 2024_
+
+We'd like to offer a big thanks to the 10 contributors who made this release possible. Here are some highlights ✨:
+
+- 🚀 Fix tooltip causing PieChart to crash on data change (#12571) @Rishi556
+- 🌍 Improve Chinese (Hong Kong) (zh-HK) and Italian (it-IT) locale for the Data Grid
+- 🐞 Bugfixes
+- 📚 Documentation improvements
+
+### Data Grid
+
+#### `@mui/x-data-grid@7.0.1`
+
+- [DataGrid] Add `resizeThrottleMs` prop (#12556) @romgrk
```

Comment:
Technically speaking this is a new feature and could justify a minor :laughing:

Reply:
OK, can change that ... makes sense